core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1552,6 +1562,7 @@ protected Scope visitTableFunctionInvocation(TableFunctionInvocation node, Optio
             }
 
             // currently we don't support input tables, so the output consists of proper columns only
+            // TODO spec p. 123
```

Comment:
Add more details -- at least mention the section title. "p. 123" is too vague (which document?) and subject to change as the document is updated